lib/firepad.js

```diff
@@ -107,6 +103,9 @@ firepad.Firepad = (function(global) {
     } else {
       this.editorAdapter_ = new MonacoAdapter(this.monaco_);
     }
+    this.editor_.adapter = this.editorAdapter_
+    this.firebaseAdapter_ = new FirebaseAdapter(ref, userId, userColor,this.editor_);
```

Comment:
I thought we decided to do it event-driven?

Reply:
Added now